src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb

```diff
@@ -68,12 +73,19 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
                 End If
             End If
 
-            Debug.Fail("Unable to get color from the shell, using a predetermined default color instead." & VB.vbCrLf & "Color Category = " & themeCategory.ToString() & ", Color Name = " & themeColorName & ", Color Type = " & colorType & ", Default Color = &h" & VB.Hex(DefaultColor.ToArgb))
+            Debug.Fail(
+$"Unable to get color from the shell, using a predetermined default color instead.
+Color Category = {themeCategory.ToString()}, Color Name = {themeColorName}, Color Type = {colorType}, Default Color = &h{ VB.Hex(defaultColor.ToArgb)}"
+)
             Return defaultColor
         End Function
 
         Private Shared Function RGBAToColor(ByVal rgbaValue As UInt32) As Color
-            Return Color.FromArgb(CInt((rgbaValue And &HFF000000UI) >> 24), CInt(rgbaValue And &HFFUI), CInt((rgbaValue And &HFF00UI) >> 8), CInt((rgbaValue And &HFF0000UI) >> 16))
+            Return Color.FromArgb(CInt((rgbaValue And &HFF000000UI) >> 24),
+                                  CInt(rgbaValue And &HFFUI),
```

Comment:
I have an ultrawide monitor, use a size 10 font and I still require scrolling to right to see the signatures, That should be the case..